client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -138,7 +138,7 @@ class StatModuleFollowers extends Component {
 								<div className="module-content-text module-content-text-stat">
 									{ wpcomData && !! wpcomData.total_wpcom && (
 										<p>
-											{ translate( 'Total WordPress.com Followers' ) }:{ ' ' }
+											{ translate( 'Total WordPress.com followers' ) }:{ ' ' }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.